Yes, this is *so* old that it predates mm-commmon.

--- origsrc/libglademm-2.6.7/configure.in	2008-09-21 08:23:11.000000000 -0500
+++ src/libglademm-2.6.7/configure.in	2013-08-02 12:23:00.256857900 -0500
@@ -143,7 +143,7 @@ Please install GNU make.]))
 
 # This macro is installed by glibmm
 # Doxygen needs the path to the installed perl.
-GLIBMM_CHECK_PERL([5.6.0])
+MM_CHECK_PERL([5.6.0])
 
 #########################################################################
 #  Dependancy checks
